### PR TITLE
TNO-2769 Add program to advanced filter

### PR DESCRIPTION
**Pull request by fbarreta**
Added program to advanced search.

Requires tno-core update.

image

### CODE REVIEW

File: libs/npm/core/src/utils/elastic/generateTextQuery.ts [hunk 1]
```diff
@@ -10,17 +10,18 @@ export const generateTextQuery = (
...
     );
   }
 
   let fields: string[] = [];
-  if (settings.inByline || settings.inStory || settings.inHeadline) {
+  if (settings.inByline || settings.inStory || settings.inHeadline || settings.inProgram) {
     if (!!settings.inByline) fields = [...fields, 'byline'];
     if (!!settings.inStory) fields = [...fields, 'summary', 'body'];
     if (!!settings.inHeadline) fields = [...fields, 'headline'];
+    if (!!settings.inProgram) fields = [...fields, 'series.name'];
```
Comment (Fosol) [inline libs/npm/core/src/utils/elastic/generateTextQuery.ts:24]: Does this actually work?  Kind amazing it's so simple to query another field.

**Review by Fosol — COMMENTED** [review]
I'll publish a `tno-core` version for you in a minute.

File: libs/npm/core/src/utils/elastic/generateTextQuery.ts (hunk 1 above)
Comment (fbarreta) [reply libs/npm/core/src/utils/elastic/generateTextQuery.ts:24]: image
Totally. Never did that, and it was so easy to implement.

**Comment by Fosol** [comment]
I've published `tno-core:0.1.90`

**Review by Fosol — APPROVED** [review]

**Pull request merged**